src/gui/mainwindow.cpp

```diff
@@ -1894,7 +1894,7 @@ void MainWindow::on_actionSearchWidget_triggered()
                 installPython();
 #else
             QMessageBox::information(this, tr("Old Python Runtime")
-                , tr("Your Python version (%1) is outdated. Please upgrade to latest version for search engines to work.\nMinimum requirement: 3.3.0.")
+                , tr("Your Python version (%1) is outdated. Please upgrade to latest version for search engines to work.\nMinimum requirement: 3.5.0.")
```

Comment:
Can you move 3.5.0. into an argument? Currently it forces a re-translation of the entire string so let's avoid that in the future.

Reply:
an example of how to go about doing this would be much appreciated.

Reply:
Thanks!

Any idea if this could be implemented https://github.com/qbittorrent/qBittorrent/pull/13268#issuecomment-917612851 or how to do so?!

Reply:
Should I also do the same for:
https://github.com/qbittorrent/qBittorrent/blob/b2e0e25f1c5a604cee23f93a227e1b31a4c07290/src/gui/mainwindow.cpp#L1890

Reply:
>Any idea if this could be implemented #13268 (comment) or how to do so?!

No idea, sorry.

>Should I also do the same for:

Sure, why not.

Reply:
>No idea, sorry.

Ok, thanks.

>Sure, why not.

Done.